solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/impl/FieldLengthFeature.java

```diff
@@ -121,11 +121,12 @@ public FieldLengthFeatureScorer(FeatureWeight weight,
 
       @Override
       public float score() throws IOException {
+        if (norms == null) {
+          return 0.0f;
+        }
 
         final long l = norms.get(itr.docID());
-        final float norm = decodeNorm(l);
-        final float numTerms = (float) Math.pow(1f / norm, 2);
-
+        final float numTerms = decodeNorm(l);
```

Comment:
should we be using a MatchAll iterator for this, of an iterator that knows all the docs which have a value for the provided field